### PR TITLE
Adding a license page to the welcome controller

### LINKED ISSUE #897: License information
The text of the license terms that users agree to when submitting items to PDC Describe is available to the public without signing in.

Approved license text [here](https://docs.google.com/document/d/1aEvbsMFJ8ZD1TEBieX6tcYO8WB4TdAtXs70r2EGrui4/edit).

Related to https://github.com/pulibrary/pdc_describe/issues/764

## Acceptance Criteria
- [ ] Create a static content page containing the text of the license
- [ ] The license page can be accessed without signing into any system

### PR DESCRIPTION
**Pull request by carolyncole**
Also change the work license to a partial so we can reuse it in welcome fixes #897

Screen Shot 2023-01-24 at 1 28 03 PM

### CODE REVIEW

File: app/views/works/_license_text.html.erb
Comment (mccalluc) [inline app/views/works/_license_text.html.erb:5]: As I understand it, this page will be linked from PDC Discovery, where it will describe the terms under which access is granted to this object. In that context, I don't think "your submission" makes sense.

I would leave this paragraph in review.html.erb, and perhaps replace it here with a different paragraph addressed to the end-user.

**Review by mccalluc — CHANGES_REQUESTED** [review]
To me, it seems like a different introduction for a different audience would make sense... but if this doesn't resonate, and someone else can approve, don't let this block.

**Review by mccalluc — APPROVED** [review]
Reconsidering... I think the default is to approve PRs to move them along, but if there are remaining questions, put them on the agenda? I'll do that.

**Comment by kelynch** [comment]
I personally think it makes sense to hold off on merging if the PR review is requesting changes to the code that we're proposing adding to the code base in that PR itself, however a follow-up review and (if appropriate) approval should then be made if/when the requested changes are made in the PR.

**Pull request merged**
